java/operator/org.eclipse.theia.cloud.operator/src/main/java/org/eclipse/theia/cloud/operator/plugins/MonitorActivityTracker.java

```diff
@@ -78,7 +80,10 @@ public void start() {
     }
 
     protected void pingAllSessions() {
-        List<Session> sessions = resourceClient.sessions().list();
+        // Only look at handled sessions (handled sessions have a lastActivity)
+        List<Session> sessions = resourceClient.sessions().list().stream()
+                .filter(session -> OperatorStatus.HANDLED.equals(session.getStatus().getOperatorStatus()))
+                .collect(Collectors.toList());
```

Comment:
Optional: We can simplify this leveraging the toList() method introduced with Java 16.
```suggestion
                .toList();
```